### DISCUSSION TITLE
Velocyto analysis with fixed cells.

### DISCUSSION THREAD

**Discussion by mbrochut** [alexdobin/STAR#2046] [Q&A]
Hello everyone!
First, thanks a lot for this wonderful tool.

I am working with Fixed RNA Profiling Probe from 10X and I am trying to do velocity analysis.

First I was doing it with the [Velocyto](https://velocyto.org/velocyto.py/) package developed by Gioele La Manno but after running it, the output file was almost empty. I then realized that, with fixed cells, the sequencing is actually the probes and not the pre-mRNA that can be immature (unsplice) or mature (splice) and so, it is actually impossible to perform velocity analysis.

Before realizing that I could not do velocity analysis, I was trying to run it with STAR and I had some problems to create the output files.

I am running this command:
```bash
STAR --soloFeatures Gene GeneFull Velocyto --readFilesIn sample_alignments.bam --genomeDir genomeDIR --readFilesType SAM SE --readFilesCommand samtools view -F 0x100 --outFileNamePrefix STAR_output_ --soloOutFileNames output_STAR/ features.tsv barcodes.tsv matrix.mtx
```
The alignment run successfully and I have a Log.final.out file but no : features.tsv, barcodes.tsv, matrix.mtx and splice/unsplice matrix.
I am using the output BAM file from CellRanger.

Is it due to my fixed sample or am I running it wrongly?

My Log.final.out:
```Started job on |	Jan 23 16:50:48
                             Started mapping on |	Jan 23 16:50:53
                                    Finished on |	Jan 23 17:19:31
       Mapping speed, Million of reads per hour |	281.29

                          Number of input reads |	134237888
                      Average input read length |	50
                                    UNIQUE READS:
                   Uniquely mapped reads number |	128540833
                        Uniquely mapped reads % |	95.76%
                          Average mapped length |	49.95
                       Number of splices: Total |	29832375
            Number of splices: Annotated (sjdb) |	29826715
                       Number of splices: GT/AG |	29706299
                       Number of splices: GC/AG |	92168
                       Number of splices: AT/AC |	29136
               Number of splices: Non-canonical |	4772
                      Mismatch rate per base, % |	0.11%
                         Deletion rate per base |	0.00%
                        Deletion average length |	1.00
                        Insertion rate per base |	0.00%
                       Insertion average length |	1.03
                             MULTI-MAPPING READS:
        Number of reads mapped to multiple loci |	5083755
             % of reads mapped to multiple loci |	3.79%
        Number of reads mapped to too many loci |	168
             % of reads mapped to too many loci |	0.00%
                                  UNMAPPED READS:
  Number of reads unmapped: too many mismatches |	0
       % of reads unmapped: too many mismatches |	0.00%
            Number of reads unmapped: too short |	613067
                 % of reads unmapped: too short |	0.46%
                Number of reads unmapped: other |	65
                     % of reads unmapped: other |	0.00%
                                  CHIMERIC READS:
                       Number of chimeric reads |	0
                            % of chimeric reads |	0.00%

```
If you could help me, I would gladly appreciate :)

ps: probes can target spliced or unspliced region so maybe we could infer some information about that.

**Comment by alexdobin** [comment]
Hi @mbrochut 

Have you tried to run it without the Velocyto option?

**Reply by mbrochut** [reply]
Hi @alexdobin Yes I also try to run it without the velocyto option but I don't have the output folder with the counts.

I only have the SAM file sorted (around 75G)

**Comment by alexdobin** [comment]
Hi @mbrochut 

what is the output in Summary.csv file?

**Reply by mbrochut** [reply]
Hi @alexdobin 

I dont have any Summary.csv file.

my files are
1. name_Alignes.out.sam
2. name_Log.final.out
3. name_Log.out,
4. name_Log.progress.out
5. name_SJ.out.tab

Maybe I am just using the wrong command and there is no output but it seems strange that the entire alignment works fine with the SAM file in output
